### PR TITLE
EN-1931 Struts2 version upgrade (v5.0.1)

### PR DESCRIPTION
**Pull request by eugeniosant**
Version upgraded from v. 2.5.10.1 to v. 2.5.17

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: admin-console/src/main/java/org/entando/entando/apsadmin/system/CustomTokenInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ * token is found: it returns the result invalid.token.
+ *
+ * TypeMessages parameter to specify the type of message to associate at
+ * invalid.token result The values of type are the following: error: return an
+ * action error message message: return an action message none: don't return
+ * message
+ *
+ * @author Entando
+ */
+public class CustomTokenInterceptor extends org.apache.struts2.interceptor.TokenInterceptor {
```
Comment (joewhite101) [inline admin-console/src/main/java/org/entando/entando/apsadmin/system/CustomTokenInterceptor.java]: This needs unit tests.

File: admin-console/src/main/java/org/entando/entando/apsadmin/system/resource/CustomLocalizedTextProvider.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import java.util.StringTokenizer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+
+/**
+ * @author E.Santoboni
+ */
+public class CustomLocalizedTextProvider extends StrutsLocalizedTextProvider {
```
Comment (joewhite101) [inline admin-console/src/main/java/org/entando/entando/apsadmin/system/resource/CustomLocalizedTextProvider.java]: Needs unit tests.

File: admin-console/src/main/java/org/entando/entando/apsadmin/system/CustomTokenInterceptor.java (hunk 1 above)
Comment (eugeniosant) [reply admin-console/src/main/java/org/entando/entando/apsadmin/system/CustomTokenInterceptor.java]: Done

File: admin-console/src/main/java/org/entando/entando/apsadmin/system/resource/CustomLocalizedTextProvider.java (hunk 2 above)
Comment (eugeniosant) [reply admin-console/src/main/java/org/entando/entando/apsadmin/system/resource/CustomLocalizedTextProvider.java]: Done

**Comment by entando-service** [comment]
All is well

**Pull request merged**